codec/fuzz/fuzz_targets/codec_roundtrip.rs

```diff
@@ -200,13 +227,14 @@ enum DecodeTarget {
     BTreeMapU8U8,
 }
 
-fn fuzz_decode_raw(data: &[u8], target: DecodeTarget, max_size: u16) {
+fn fuzz_decode_raw(data: &[u8], target: DecodeTarget, min_size: u16, max_size: u16) {
     if data.len() > MAX_INPUT_SIZE {
         return;
     }
 
-    let max_size = (max_size as usize).min(MAX_COLLECTION_SIZE);
-    let range_cfg: RangeCfg = (0..=max_size).into();
+    let min_size = (min_size as usize).clamp(MIN_COLLECTION_SIZE, MAX_COLLECTION_SIZE);
+    let max_size = (max_size as usize).clamp(MIN_COLLECTION_SIZE, MAX_COLLECTION_SIZE);
+    let range_cfg: RangeCfg = (min_size..=max_size).into();
```

Comment:
since `min_size < max_size` is useless, I wonder if it's worth clamping max_size with a lower bound of `min_size` rather than `MIN_COLLECTION_SIZE`. Because we aren't here to fuzz the `Range` trait in rust (though I guess you could argue we are sort of fuzzing `Into<RangeCfg>`, I don't think it's that important